packages/MarkdownEditor-Tests.package/MarkdownInlineTextStylerTest.class/instance/testPreviousAttributesGetRemoved.st

```diff
@@ -1,10 +1,10 @@
 tests - parsing
-testPlainStylingKeepsAttributes
+testPreviousAttributesGetRemoved
```

Comment:
The `BlockTextStyler` should also remove attributes. Could you add a test for it?